doc/architecture/decisions/0017-mutual-tls.md

```diff
@@ -0,0 +1,46 @@
+# 17. mutual TLS (mTLS)
+[Next >>](9999-end.md)
+
+Date: 2023-02-28
+
+## Status
+
+Proposal for discussion
+
+## Context
+
+Some acquirers and data suppliers have a desire for additional security measures over and above encryption in transit
+and the OIDC authentication currently in place, as part of a Zero Trust approach.
```

Comment:
I'm not sure we can currently claim a Zero Trust approach. Perhaps we should, but then mTLS termination at the gateway might not be a good fit - we would certainly need some encryption in transit over the VPC

Reply:
I think I was using Zero Trust here as a "it's possible departments are moving towards this so would want all their touch points to do so"